src/oatpp/core/utils/ConversionUtils.cpp

```diff
@@ -189,11 +190,30 @@ namespace oatpp { namespace utils { namespace conversion {
   }
 
   v_buff_size float32ToCharSequence(v_float32 value, p_char8 data, v_buff_size n, const char* format) {
-    return snprintf((char*)data, n, format, value);
+    return float64ToCharSequence(value, data, n, format);
   }
 
-v_buff_size float64ToCharSequence(v_float64 value, p_char8 data, v_buff_size n, const char* format) {
-    return snprintf((char*)data, n, format, value);
+  v_buff_size float64ToCharSequence(v_float64 value, p_char8 data, v_buff_size n, const char* format) {
```

Comment:
Please do not change the behavior of this method.
Instead, please add a new method to json::Utils
https://github.com/oatpp/oatpp/blob/master/src/oatpp/parser/json/Utils.hpp